pkg/owner/manager.go

```diff
@@ -499,3 +499,31 @@ func init() {
 		logutil.BgLogger().Warn("set manager session TTL failed", zap.Error(err))
 	}
 }
+
+// AcquireDistributedLock creates a mutex with ETCD client, and returns a mutex release function.
+func AcquireDistributedLock(
+	ctx context.Context,
+	cli *clientv3.Client,
+	key string,
+	ttlInSec int,
+) (release func(), err error) {
+	se, _ := concurrency.NewSession(cli, concurrency.WithTTL(ttlInSec))
+	mu := concurrency.NewMutex(se, key)
+	err = mu.Lock(ctx)
+	if err != nil {
```

Comment:
the session is not closed

Reply:
https://github.com/pingcap/tidb/pull/56184/commits/5f4d438b232019de96a10495d721b8fd90d3eb5c